tools/distributions/debian/bazel_debian_license

```diff
@@ -0,0 +1,2 @@
+Please find the license file of Bazel at /usr/share/doc/bazel/copyright
```

Comment:
How is `/usr/share/doc/bazel/copyright` created?

Reply:
By hand, unfortunately. Although we do have [some tools](https://packages.debian.org/sid/licensecheck) to make it (slightly) less painful.

Reply:
When I started working on Debian I had no idea how much copyright law was involved! ;) Or how important it is...